server/src/main/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceComputer.java

```diff
@@ -403,24 +403,29 @@ public DesiredBalance compute(
                         convergenceLogMsgLevel,
                         () -> Strings.format(
                             """
-                                Desired balance computation for [%d] converged after [%s] and [%d] iterations, \
-                                resumed computation [%d] times with [%d] iterations since the last resumption [%s] ago""",
+                                Desired balance computation for [%d] converged after [%s] and [%d] iterations this round, \
+                                resumed computation [%s] ago with [%d] iterations over [%d] rounds since the last convergence \
+                                [%s] ago""",
                             desiredBalanceInput.index(),
+                            TimeValue.timeValueMillis(currentTime - computationStartedTime).toString(),
+                            iterations,
                             TimeValue.timeValueMillis(currentTime - firstComputeStartedSinceConvergedTimeMillis).toString(),
                             numIterationsSinceLastConverged,
                             numComputeCallsSinceLastConverged,
-                            iterations,
-                            TimeValue.timeValueMillis(currentTime - computationStartedTime).toString()
+                            TimeValue.timeValueMillis(currentTime - lastConvergedTimeMillis).toString()
                         )
                     );
                 } else {
                     logger.log(
                         convergenceLogMsgLevel,
                         () -> Strings.format(
-                            "Desired balance computation for [%d] converged after [%s] and [%d] iterations",
+                            """
+                                Desired balance computation for [%d] converged after [%s] and [%d] iterations this round, \
```

Comment:
"this round" implies there have been other rounds, but that's not the case here. Maybe "in a single round" would be clearer?